shared/src/test/scala/mlscript/DiffTests.scala

```diff
@@ -359,7 +453,72 @@ class DiffTests extends org.scalatest.funsuite.AnyFunSuite {
     }
     if (testFaield)
       fail(s"Unexpected diagnostics (or lack thereof) at: " + failures.map("l."+_).mkString(", "))
-    
+    host.terminate()
```

Comment:
This should probably be written as:

```scala
val host = ReplHost()
try {

  ...

} finally host.terminate()
```

to avoid leaking processes.